ceph/object-storage/nano/base/service.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ceph-nano
+  labels:
+    app: ceph
+    daemon: nano
+spec:
+  ports:
+  - name: cn-s3
+    port: 80
+    protocol: TCP
+    targetPort: 8000
+  selector:
+    app: ceph
+    daemon: nano
+---
+# This service is a workaround for the fact that cn-core hardcodes the radosgw to pod hostname
+# With this any pod in the namespace can set the s3 endpoint to http://ceph-nano-0
+apiVersion: v1
+kind: Service
+metadata:
+  name: ceph-nano-0
+  labels:
+    app: ceph
+    daemon: nano
+spec:
+  ports:
+  - name: cn-s3
+    port: 80
+    protocol: TCP
+    targetPort: 8000
+  selector:
+    app: ceph
+    daemon: nano
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: ceph-nano-dashboard
+  labels:
+    app: ceph
+    daemon: nano
+spec:
+  ports:
+  - name: cn-sree
+    port: 5001
+    protocol: TCP
+    targetPort: 5001
+  selector:
+    app: ceph
+    daemon: nano
```

Comment:
Is there a prometheus metrics port?

Reply:
It doesn't appear that metrics are enabled by default (if available)